server/app/services/question/types/QuestionType.java

```diff
@@ -34,7 +35,8 @@ public enum QuestionType {
   RADIO_BUTTON("Radio Button", SingleSelectQuestion.class),
   STATIC("Static Text", StaticContentQuestion.class),
   TEXT("Text Field", TextQuestion.class),
-  PHONE("Phone Field", PhoneQuestion.class);
+  PHONE("Phone Field", PhoneQuestion.class),
+  NULL_QUESTION("Null Field", NullQuestion.class);
```

Comment:
**suggestion (blocking):** it's not a really a "field" since it has no UI. maybe this?
```suggestion
  NULL_QUESTION("missing question", NullQuestion.class);
```